lib/ClangImporter/ClangImporter.cpp

```diff
@@ -3832,8 +3837,23 @@ void ClangImporter::Implementation::lookupValue(
     ValueDecl *decl;
     // If it's a Clang declaration, try to import it.
     if (auto clangDecl = entry.dyn_cast<clang::NamedDecl *>()) {
-      decl = cast_or_null<ValueDecl>(
-          importDeclReal(clangDecl->getMostRecentDecl(), CurrentVersion));
+      // When looking for a namespace, we need to populate all of its members.
+      // To do this, let's import all of its redeclarations.
+      if (auto ns = dyn_cast<clang::NamespaceDecl>(clangDecl)) {
+        Decl *realDecl;
```

Comment:
I know this is always going to be set before it's read, but can you still initialize it to `nullptr`? I've spent too much time debugging things that were caused by a default initialized pointer that should have been zero-initialized. It's too easy for someone to refactor/change this code in the future without realizing that they have to update this line too.

Reply:
Yes you're totally right, thanks for noticing!